cd/build_images.sh

```diff
@@ -19,13 +19,17 @@ set -e
 root_directory="$( cd $(dirname $(dirname $0)) > /dev/null 2>&1 && pwd )"
 cd $root_directory
 
+if [ ! -z "$1" ]; then
+	ENV_FILE=$1
+fi
```

Comment:
If the argument isn't set then the script will proceed anyway. It might be better to exit with an error message.